lib/network_utils.pm

```diff
@@ -137,4 +137,22 @@ sub ifc_is_up {
     return !script_run("ip link show dev $ifc | grep 'state UP'");
 }
 
+=head2 genmac
+
+Generate custom MAC address.
+Used for Xen domU testing, to define MAC address once for whole test suite lifecycle.
+
+ genmac(['aa:bb:cc'])
+
+=cut
+
+sub genmac {
+    my @mac = split(/:/, shift);
```

Comment:
```suggestion
    my @mac = split(/:/, shift) // "";
```

In case someone wants a MAC address without a given prefix?

Reply:
Actually, I think we would mix Perl context as well over here. If we leave it as it was, it would be either list context on `undef`. In the other case, with assigning an empty string I am afraid that we would be very close to scalar context.